docs/web3.eth.rst

```diff
@@ -992,7 +992,11 @@ The following methods are available on the ``web3.eth`` namespace.
 
     * Delegates to ``eth_createAccessList`` RPC Method
 
-    This method creates an `EIP2930 <https://eips.ethereum.org/EIPS/eip-2930>`_ type ``accessList`` based on a given ``Transaction``. The ``accessList`` contains all storage slots and addresses read and written by the transaction, except for the sender account and the precompiles. This method uses the same ``transaction`` call object and ``block_identifier`` object as :meth:`~web3.eth.Eth.call()`. An ``accessList`` can be used to unstuck contracts that became inaccessible due to gas cost increases.
+    This method creates an `EIP-2930 <https://eips.ethereum.org/EIPS/eip-2930>`_ type ``accessList`` based on a given
+    ``Transaction``. The ``accessList`` contains all storage slots and addresses read and written by the transaction,
+    except for the sender account and the precompiles. This method uses the same ``transaction`` call object and
+    ``block_identifier`` object as :meth:`~web3.eth.Eth.call()`. An ``accessList`` can be used to unstick contracts that
```

Comment:
Just a nit. I see they use `"unstuck"` in quotes in the EIP, pertaining to one of the past EIPs that gave rise to these issues. Even in the eip it's sort of ambiguous language. Take or leave.

```suggestion
    ``block_identifier`` object as :meth:`~web3.eth.Eth.call()`. An ``accessList`` can be used to access contracts that
```